apps/chat/src/components/Chat/ReportIssueDialog.tsx

```diff
@@ -72,6 +67,15 @@ export const ReportIssueDialog: FC<Props> = ({ isOpen, onClose }) => {
     [],
   );
 
+  useEffect(() => {
+    if (isSuccessfullySent) {
+      dispatch(ServiceActions.resetIsSuccessfullySent());
+
+      setTitle('');
```

Comment:
Should we close the dialog after sending?

Reply:
It closes in handleSubmit method